examples/run_quickstart_new_config.py

```diff
@@ -0,0 +1,156 @@
+        folds=folds,
+        methods=methods,
+        ignore_cache=ignore_cache,
+    )
+
+    # Convert the run artifacts into an EvaluationRepository
+    repo: EvaluationRepository = exp_batch_runner.repo_from_results(results_lst=results_lst)
+
+    repo.print_info()
+
+    save_path = "repo_quickstart_new_config"
+    repo.to_dir(path=save_path)  # Load the repo later via `EvaluationRepository.from_dir(save_path)`
+
+    new_baselines = repo.baselines()
+    new_configs = repo.configs()
+    print(f"New Baselines : {new_baselines}")
+    print(f"New Configs   : {new_configs}")
+    print(f"New Configs Hyperparameters: {repo.configs_hyperparameters()}")
+
+    shared_datasets = [d for d in repo.datasets(union=False) if d in repo_og.datasets()]
+    shared_folds = [f for f in repo.folds if f in repo_og.folds]
+    repo_combined = EvaluationRepositoryCollection(repos=[repo_og, repo], config_fallback="ExtraTrees_c1_BAG_L1")
+
+    # subset repo_combined to only contain datasets and folds shared by both repo and repo_og
+    repo_combined = repo_combined.subset(datasets=shared_datasets, folds=shared_folds)
+
+    repo_combined.print_info()
+
+    comparison_configs_og = [
+        "RandomForest_c1_BAG_L1",
+        "ExtraTrees_c1_BAG_L1",
+        "LightGBM_c1_BAG_L1",
+        "XGBoost_c1_BAG_L1",
+        "CatBoost_c1_BAG_L1",
+        "NeuralNetTorch_c1_BAG_L1",
+        "NeuralNetFastAI_c1_BAG_L1",
+    ]
+
+    # Include our new configs
+    comparison_configs = comparison_configs_og + new_configs
+
+    # Baselines to compare configs with
+    # baselines = repo_combined.baselines(union=False)  # to compare with all baselines
+    baselines = [
+        "AutoGluon_bq_4h8c_2023_11_14",
+        "H2OAutoML_4h8c_2023_11_14",
+        "autosklearn2_4h8c_2023_11_14",
+        "flaml_4h8c_2023_11_14",
+        "lightautoml_4h8c_2023_11_14",
+    ]
+    baselines += new_baselines
+
+    # create an evaluator to compute comparison metrics such as win-rate and ELO
+    evaluator = Evaluator(repo=repo_combined)
+    metrics = evaluator.compare_metrics(
+        datasets=datasets,
+        folds=folds,
+        baselines=baselines,
+        configs=comparison_configs,
+    )
+
+    metrics_tmp = metrics.reset_index(drop=False)
+
+    with pd.option_context("display.max_rows", None, "display.max_columns", None, "display.width", 1000):
+        print(f"Config Metrics Example:\n{metrics.head(100)}")
+
+    # Requires `autogluon_bench` and `autogluon_benchmark`
+    # FIXME: AG-Bench is saving unnecessary files in `data/results/output/openml/None/...`
+    # FIXME: AG-Bench might not work out of the box due to the existence of `run_evaluation_openml.evaluate_amlb_results`
+    evaluation_save_dir = os.path.join(expname, "evaluation")
+    evaluator_output = evaluator.plot_overall_rank_comparison(
+        results_df=metrics,
+        save_dir=evaluation_save_dir,
+        evaluator_kwargs={"treat_folds_as_datasets": True},
+    )
```

Comment:
Could we try import here? It seems like we dont want to fail the run if this table is not plotted since we still have one above and the dependency is optional?

Reply:
I'm leaving this for a future PR, as I eventually want to remove the dependency on AG-Bench and AG-Benchmark by moving the required logic into TabRepo directly. TabRepo should ideally replace AG-Benchmark, as AG-Benchmark was my old code for running experiments prior to making TabRepo 2.0.